tools/ci_build/build.py

```diff
@@ -1200,6 +1200,22 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                 [sys.executable, 'onnxruntime_test_python.py'],
                 cwd=cwd, dll_path=dll_path)
 
+            iobinding_test = False
```

Comment:
Make sure tests are run in atleast one build pipeline. This should now run in Linux GPU CI pipeline.